libs/utils/android/workload.py

```diff
@@ -62,7 +62,7 @@ def _check_availables(cls, test_env):
         _log.debug('Building list of available workloads...')
         for sc in Workload.__subclasses__():
             _log.debug('Checking workload [%s]...', sc.__name__)
-            if sc.package in cls._packages:
+            if sc.package in cls._packages or sc.package == '':
```

Comment:
We can avoid this check by initialising `cls._packages` to always include the empty string.